tests/test_series.py

```diff
@@ -3276,18 +3276,34 @@ def test_map_na() -> None:
 
     mapping = {1: "a", 2: "b", 3: "c"}
     check(assert_type(s.map(mapping, na_action=None), "pd.Series[str]"), pd.Series, str)
+    check(assert_type(s.map(mapping), "pd.Series[str]"), pd.Series, str)
 
     def callable(x: int | NAType) -> str | NAType:
         if isinstance(x, int):
             return str(x)
         return x
 
+    def bad_callable(x: int) -> int:
+        return x + 1
+
+    s.map(
+        bad_callable, na_action=None  # type: ignore[arg-type] # pyright: ignore[reportCallIssue, reportArgumentType]
+    )
+    s.map(bad_callable)  # type: ignore[arg-type] # pyright: ignore[reportArgumentType]
```

Comment:
The type checkers should accept those patterns, because at least in my tests at runtime, no error is raised.  So why are you considering that a "bad callable"?

Reply:
Ah, may bad. So normally if `na_action=None` then the function has to be able to handle `pd.NA` because it gets passed in. And for my `bad_callable` it does not explicitely handle it, at least according to the type hint.

I just chose a poor operation, to perform in the function, because pd.NA supports addition of an integer.

I have now changed it for a different operation on integers that pd.NA does not incidentally support.